### ISSUE TITLE
checking NARR source for similar wind bias

### ISSUE THREAD

**Issue by jread-usgs** [USGS-R/mda.lakes#72]
related to #70 

These files are broken up into yearly directories, and the temporal resolution is coarser than NLDAS-2 (3 hourly and ~32 km)

**Comment by jread-usgs** [comment]
From Mitchell ea 2004: "To date, retrospective forcing is available from 1 October 1996 through 2002. Real-time forcing is available from 16 April 1999 to present."

**Comment by lawinslow** [comment]
Hmm, pretty close timing to the jump we see.

**Comment by jread-usgs** [comment]
![image](https://cloud.githubusercontent.com/assets/2349007/8625924/32d2536a-2707-11e5-8f52-476988b8e5c1.png)

**Comment by jread-usgs** [comment]
@lawinslow could that jump be pinned at the start of 2002, or the start of 2001?

**Comment by lawinslow** [comment]
Yeah, though I call it 2001, its really the *start* of 2002 (`2001-12-31`)

**Comment by jread-usgs** [comment]
ok, so that would exactly line up with a full switch to real-time forcing...

**Comment by lawinslow** [comment]
smoking gun...

**Comment by jread-usgs** [comment]
no mention here: http://www.emc.ncep.noaa.gov/mmb/rreanl/faq.html

**Comment by jread-usgs** [comment]
In NARR, wind obs comes from: 
Rawinsondes, dropsondes, pibals, aircraft, and geostationary satellite cloud drift movements

**Comment by lawinslow** [comment]
Only the cool kids use dropsondes

**Comment by jread-usgs** [comment]
Not quite done with this, but I am not seeing the same thing... 
![image](https://cloud.githubusercontent.com/assets/2349007/8628275/35304184-2717-11e5-854e-bc93c958f2b8.png)

**Comment by jread-usgs** [comment]
![image](https://cloud.githubusercontent.com/assets/2349007/8631092/e148003e-2734-11e5-81f1-e95da7a0c994.png)

**Comment by jread-usgs** [comment]
there we go @lawinslow

**Comment by lawinslow** [comment]
Yup, mystery solved.

**Comment by lawinslow** [comment]
Also good to know it persists up to today. I had only looked through 2013.

**Comment by jread-usgs** [comment]
yeah, I skipped 2015 because it is/was incomplete as a season
